homeassistant/components/rainmachine/binary_sensor.py

```diff
@@ -6,35 +6,47 @@
 from homeassistant.helpers.dispatcher import async_dispatcher_connect
 
 from . import (
-    BINARY_SENSORS,
     DATA_CLIENT,
     DOMAIN as RAINMACHINE_DOMAIN,
     PROVISION_SETTINGS,
     RESTRICTIONS_CURRENT,
     RESTRICTIONS_UNIVERSAL,
     SENSOR_UPDATE_TOPIC,
-    TYPE_FLOW_SENSOR,
-    TYPE_FREEZE,
-    TYPE_FREEZE_PROTECTION,
-    TYPE_HOT_DAYS,
-    TYPE_HOURLY,
-    TYPE_MONTH,
-    TYPE_RAINDELAY,
-    TYPE_RAINSENSOR,
-    TYPE_WEEKDAY,
     RainMachineEntity,
 )
 
 _LOGGER = logging.getLogger(__name__)
 
+TYPE_FLOW_SENSOR = "flow_sensor"
+TYPE_FREEZE = "freeze"
+TYPE_FREEZE_PROTECTION = "freeze_protection"
+TYPE_HOT_DAYS = "extra_water_on_hot_days"
+TYPE_HOURLY = "hourly"
+TYPE_MONTH = "month"
+TYPE_RAINDELAY = "raindelay"
+TYPE_RAINSENSOR = "rainsensor"
+TYPE_WEEKDAY = "weekday"
+
+BINARY_SENSORS = {
```

Comment:
Are all binary sensors interesting for users? Or should we use the `entity_registry_enabled_default` property to disable some entities by default? (ie weekday)

Reply:
Great point. Let me think through which ones would be useful to all users.

Reply:
14f3488